### DISCUSSION TITLE
Is it possible to limit the run/eval batch size?

### DISCUSSION THREAD

**Discussion by ebotiab** [microsoft/promptflow#3579] [Q&A] [answered]
When making a run is it possible to limit the quantity of the data that is passed to be executed at once?

**Comment by 0mza987** [comment, marked as answer]
Thanks for the feedback, I have created a feature request accordingly: https://github.com/microsoft/promptflow/issues/3580

Due to the bandwidth issue, it won't be implemented right away, so we've labeled it as a long-term task. We hope to address it in the future as our resources allow.
